src/main/java/com/redhat/devtools/lsp4ij/console/actions/ApplyLanguageServerSettingsAction.java

```diff
@@ -41,11 +59,82 @@ public void actionPerformed(@NotNull AnActionEvent e) {
 
     @Override
     public void update(@NotNull AnActionEvent e) {
-        e.getPresentation().setEnabled(languageServerView.isModified());
+        boolean modified = languageServerView.isModified();
+        var project = e.getProject();
+
+        if (project != null) {
+            boolean showSaveTipOnConfigurationChange = com.redhat.devtools.lsp4ij.settings.UserDefinedLanguageServerSettings.getInstance(project).showSaveTipOnConfigurationChange();
+            // Only show the tooltip once per configuration change
+            if (modified && !hasSaveTipBalloonShown && showSaveTipOnConfigurationChange) {
+                hasSaveTipBalloonShown = true;
+                showBalloon(project);
+            }
+            // If configuration is returned to match the unmodified state, reset the tooltip to be shown again
+            if (!modified) {
+                hasSaveTipBalloonShown = false;
+            }
+        }
+
+        e.getPresentation().setEnabled(modified);
     }
 
     @Override
     public @NotNull ActionUpdateThread getActionUpdateThread() {
         return ActionUpdateThread.BGT;
     }
+
+    /**
+     * Create and show a balloon to draw attention to the save button when modifying
+     * configurations in the LSP console
+     * @param project passed on to the panel creation to create the button's action listener
+     */
+    private void showBalloon(@NotNull Project project) {
+        var jbPanel = createBalloonPanel(project);
+
+        BalloonBuilder builder = JBPopupFactory.getInstance()
+                .createBalloonBuilder(jbPanel)
+                .setFadeoutTime(1600) // How many ms the balloon is shown for

```

Comment:
IMHO, The fade out is not enough, I will set to `10000` to give some times to the user to see the tip.